packages/react-native-windows-init/README.md

```diff
@@ -15,9 +15,16 @@ npx react-native-windows-init --language cpp
 Options:
```

Comment:
We publish react-native-windows-init from master. No need to backport anything to it (we make the version in stable branches a private/unpublished package).

Reply:
@asklar: Sorry should have known this..

Reply:
should I undo the changes in the init package or keep them to align w/what's in master?

Reply:
@NickGerleman After more thinking I'm confused...
Does this mean that we only publish on major releases (i.e. 0.64) or daily from the 0.0.0-canary.xyz?

Reply:
ah wow!!! It seems daily... https://www.npmjs.com/package/react-native-windows-init

I suspect then that the public CLI on npm MUST be  broken now as it has a new dependency as it now depends on the telemetry package which is NOT published daily....

Reply:
Okay... it does seem to work... but it picks up telemetry with version: "@react-native-windows/telemetry": "^0.0.0-canary.2",

Reply:
Yep, init is published daily, for better or worse. It also needs to support prior versions of RNW, which can make us only testing it in master problematic sometimes.